lib/convection/model/template/resource/aws_elbv2_listener.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../resource'
+
+module Convection
+  module Model
+    class Template
+      class Resource
+        ##
+        # AWS::ElasticLoadBalancingV2::Listener
+        ##
+        class ELBV2Listener < Resource
+          type 'AWS::ElasticLoadBalancingV2::Listener', :elbv2_listener
+          property :certificates, 'Certificates', :type => :list
+          property :default_actions, 'DefaultActions', :type => :list
```

Comment:
Fixed in a later commit.  The singular is now a function, which is consistent with code elsewhere.